atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -408,16 +409,16 @@ public void disableReadWriteConflictChecking(TableReference tableRef) {
         hasReads = true;
         Map<byte[], RowColumnRangeIterator> rawResults = keyValueService.getRowsColumnRange(tableRef, rows,
                 columnRangeSelection, getStartTimestamp());
-        Map<byte[], Iterator<Map.Entry<Cell, byte[]>>> postFilteredResults =
-                Maps.newHashMapWithExpectedSize(rawResults.size());
+        ImmutableSortedMap.Builder<byte[], Iterator<Map.Entry<Cell, byte[]>>> postFilteredResults =
+                ImmutableSortedMap.orderedBy(UnsignedBytes.lexicographicalComparator());
```

Comment:
nit: I think in other places we use PtBytes#BYTES_COMPARATOR shim to this method. Just something I noticed.